tests/Gaufrette/Functional/Adapter/AzureMultiContainerBlobStorageTest.php

```diff
@@ -98,6 +98,19 @@ public function shouldGetMtime()
     /**
      * @test
      * @group functional
+     */
+    public function shouldGetSize()
+    {
+        $path = $this->createUniqueContainerName('container') . '/foo';
+
+        $this->filesystem->write($path, 'Some content');
+
+        $this->assertGreaterThan(11, $this->filesystem->size($path));
```

Comment:
You should assert about the exact size of the file.

Reply:
Thanks, updated the testcode.